wordpress-mobile/WordPress-iOS
# For testing image size setting, click to cycle though available sizes.

### Fixes [gb-mobile 1452](https://github.com/wordpress-mobile/gutenberg-mobile/issues/1452#issuecomment-562033000)

### Description
This PR is for testing a simple approach to updating Image Sizes from the Image Block Settings menu by cycling through available values when the size row is pressed. 

![size-changes](https://user-images.githubusercontent.com/1103838/71149858-da3b8e00-21fd-11ea-9090-e44eacb2450a.gif)

### To test:
1. Navigate to the Block Editor
2. Add an Image Block and upload an image
3. Click the gear for block settings
4. Tap Image Size row and verify value cycles through available sizes (Thumbnail, Medium, Large, Fullscreen).
5. Verify the Image size changes as you click through the menu.
6. Verify the Image Size is preserved if you update the post, leave, and return. 


PR submission checklist:
